Common/OpenCL/Factories/itkGPUResampleImageFilterFactory.h

```diff
@@ -73,89 +73,164 @@ class ITK_TEMPLATE_EXPORT GPUResampleImageFilterFactory2 : public GPUObjectFacto
+      typeid(GPUResampleImageFilter<InputImageType, GPUOutputImageType, float, float>).name(),
+      "GPU ResampleImageFilter override GPUImage second, interpolator float and transform double",
+      true,
+      CreateObjectFunction<GPUResampleImageFilter<InputImageType, GPUOutputImageType, float, float>>::New());
+
+    // Override when itkGPUImage is second template argument
+    // With interpolator precision type as double and transform as float, on GPU as float
+    this->RegisterOverride(
+      typeid(ResampleImageFilter<InputImageType, GPUOutputImageType, double, float>).name(),
+      typeid(GPUResampleImageFilter<InputImageType, GPUOutputImageType, float, float>).name(),
+      "GPU ResampleImageFilter override GPUImage second, interpolator double and transform float",
+      true,
+      CreateObjectFunction<GPUResampleImageFilter<InputImageType, GPUOutputImageType, float, float>>::New());
 
     // Override when itkGPUImage is first and second template arguments
     this->RegisterOverride(
-      typeid(ResampleImageFilter<GPUInputImageType, GPUOutputImageType, float>).name(),
-      typeid(GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float>).name(),
-      "GPU ResampleImageFilter override GPUImage first and second, interpolator float",
+      typeid(ResampleImageFilter<GPUInputImageType, GPUOutputImageType, float, float>).name(),
+      typeid(GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float, float>).name(),
+      "GPU ResampleImageFilter override GPUImage first and second, interpolator float and transform float",
       true,
-      CreateObjectFunction<GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float>>::New());
+      CreateObjectFunction<GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float, float>>::New());
 
     // Override when itkGPUImage is first and second template arguments
-    // With interpolator precision type as double, on GPU as float
+    // With interpolator precision type as double and transform double, on GPU as float
     this->RegisterOverride(
-      typeid(ResampleImageFilter<GPUInputImageType, GPUOutputImageType, double>).name(),
-      typeid(GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float>).name(),
-      "GPU ResampleImageFilter override GPUImage first and second, interpolator double",
+      typeid(ResampleImageFilter<GPUInputImageType, GPUOutputImageType, double, double>).name(),
+      typeid(GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float, float>).name(),
+      "GPU ResampleImageFilter override GPUImage first and second, interpolator double and transform double",
       true,
-      CreateObjectFunction<GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float>>::New());
+      CreateObjectFunction<GPUResampleImageFilter<GPUInputImageType, GPUOutputImageType, float, float>>::New());
   }
```

Comment:
This `}` should be moved down, beyond the last `this->RegisterOverride` call!

Reply:
@ntatsisk  I see, you fixed those `}` as well, by https://github.com/SuperElastix/elastix/compare/86468c87bef8c29cf7236159cff91d232fb87680..5901d51a30c5a67604a5081287a282e2075c9eff  👍